openmoc/openmoc.i

```diff
@@ -64,8 +67,11 @@
 %ignore setFSRsToMaterialIDs(std::vector<int>* FSRs_to_material_IDs);
 %ignore setFSRKeysMap(ParallelHashMap<std::size_t, fsr_data*>* FSR_keys_map);
 %ignore initializeFSRVectors();
-
-
+%ignore getArray();
```

Comment:
Perhaps add a comment here as to why SWIG is being instructed to ignore these methods?